third-party-oauth/shared/utils/o-auth.ts

```diff
@@ -1,12 +1,23 @@
-import { signJwt, OAuthSourceType } from '../mod.ts'
+import { OAuthSourceType, signJwt } from "../mod.ts";
 
-const REDIRECT_URI = window?._truffleInitialData?.clientConfig?.IS_PROD_ENV ? "https://third-party-oauth.truffle.vip/redirect/callback" : "http://localhost:50230/redirect/callback";
+const REDIRECT_URI = window?._truffleInitialData?.clientConfig?.IS_PROD_ENV
+  ? "https://third-party-oauth.truffle.vip/redirect/callback"
+  : window?._truffleInitialData?.clientConfig?.IS_STAGING_ENV
+  ? "https://platform-third-party-oauth.sporocarp.dev/redirect/callback"
+  : "http://localhost:50230/redirect/callback";
 const AUTHORIZE_URL = "https://accounts.google.com/o/oauth2/v2/auth";
 const SCOPES = ["https://www.googleapis.com/auth/youtube.readonly"];
-const GOOGLE_CLIENT_ID = "710684531554-58ngiqv3jreg88ns30s3vkptb401rja1.apps.googleusercontent.com";
 
-export const getYouTubeOAuthUrl = async (accessToken: string, orgId: string) => {
-  const stateStr = await signJwt('youtube', accessToken, orgId);
+const GOOGLE_CLIENT_ID =
+  window?._truffleInitialData?.clientConfig?.IS_STAGING_ENV
```

Comment:
Can you add a hack / todo comment here to make a standard way of detecting dev vs staging vs prod? No packages should be using window._truffleInitialData directly

We can figure out right solution later, not now